testsuite/features/step_definitions/command_steps.rb

```diff
@@ -5,6 +5,25 @@
 require 'timeout'
 require 'nokogiri'
 
+When(/^I delete these channels with spacewalk\-remove\-channel:$/) do |table|
+  channels_cmd = "spacewalk-remove-channel "
+  table.raw.each { |x| channels_cmd = channels_cmd + " -c " + x[0] }
+  puts channels_cmd
+  # out, _code = $server.run("spacewalk-remove-channel -l")
+  @command_output, return_code = $server.run(channels_cmd, false)
+  puts return_code
```

Comment:
Having some logs to debug is nice, but for automating validation, better if we do some check with return_code and @command_output

Reply:
I cannot validate anything in the return_code or command output in the point of the code

Reply:
Then why we need to print those logs?

Reply:
You are right. I just remove it. Thanks